app/utils.py

```diff
@@ -3,6 +3,19 @@
 from six import iteritems, string_types
 from werkzeug.exceptions import BadRequest
 
+from .validation import validate_updater_json_or_400
+
+
+def validate_and_return_updater_request():
+    json_payload = get_json_from_request()
+
+    if 'update_details' in json_payload:
+        json_payload = json_payload['update_details']
+
+    validate_updater_json_or_400(json_payload)
+
+    return {'updated_by': json_payload['updated_by']}
```

Comment:
Can we not just `return json_payload` here? I don't think `update_details` ever has any other keys now? (`update_reason` got killed a long time ago, didn't it?)